config/webpack.config.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
                 options: {
                     presets: [
                         'env',
+                        'stage-0',
```

Comment:
I think if we had the following, we would have all of the pieces from `stage-0` and `es2015` that we need:

```
{
 "presets": [
  ["env",{
    "targets": {
        "browsers": [
            "> 1%",
            "last 2 versions"
          ],
        },
        "loose": true,
        "useBuiltIns": true
      ],
      "react"
 ],
 "plugins": [
    "transform-object-rest-spread",
    "transform-class-properties"
  ]
}
```